inji-verify/src/utils/qr-utils.js

```diff
@@ -1,11 +1,16 @@
 import { scanFile } from "@openhealthnz-credentials/pdf-image-qr-scanner";
```

Comment:
This is unused now. We can remove `@openhealthnz-credentials/pdf-image-qr-scanner` as started using `qr-scanner`